packages/discovery-provider/ddl/migrations/0053_playlists_tracks_relations.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS collection_track_relations (
```

Comment:
i dont see "_relations" as a pattern with other tables, not sure if there is one though. mabye `playlist_tracks` or `playlist_track_relation`?

Reply:
def no pre-existing pattern, tho other table names are plural eg `tracks` and `playlists`. inclined to just leave this as is but lmk if you have strong feelings.